main-actions/src/main/scala/sbt/Pkg.scala

```diff
@@ -0,0 +1,338 @@
+/*
+ * sbt
+ * Copyright 2011 - 2018, Lightbend, Inc.
+ * Copyright 2008 - 2010, Mark Harrah
+ * Licensed under Apache License 2.0 (see LICENSE)
+ */
+
+package sbt
+
+import java.io.File
+import java.time.OffsetDateTime
+import java.util.jar.{ Attributes, Manifest }
+import scala.collection.JavaConverters._
+import sbt.io.IO
+
+import sjsonnew.{
+  :*:,
+  Builder,
+  IsoLList,
+  JsonFormat,
+  LList,
+  LNil,
+  Unbuilder,
+  deserializationError,
+  flatUnionFormat4
+}
+
+import sbt.util.Logger
+import sbt.util.{ CacheStoreFactory, FilesInfo, ModifiedFileInfo, PlainFileInfo }
+import sbt.util.FileInfo.{ exists, lastModified }
+import sbt.util.CacheImplicits._
+import sbt.util.Tracked.{ inputChanged, outputChanged }
+import scala.sys.process.Process
+import xsbti.{ FileConverter, HashedVirtualFileRef, VirtualFile, VirtualFileRef }
+
+/**
+ * == Package ==
+ *
+ * This module provides an API to package jar files.
+ *
+ * @see [[https://docs.oracle.com/javase/tutorial/deployment/jar/index.html]]
+ */
+object Pkg:
```

Comment:
OS like macOS and Windows typically use case insensitive file system, so `Package` as a class name is a bad idea since it can't coexist with a package object artifacts like class file or `package.html` file for Scaladoc.